Model/Feed.php

```diff
@@ -203,6 +214,8 @@ public function sendProducts($pageSize = 50)
         }
 
         $this->logger->debug("PureClarity: Finished sending product data");
+
```

Comment:
try/catch to ensure we always end emulation regardless of what happens in the export?